### PR TITLE
fix(SQFormGuidedWorkflow.js):  add style prop for container of guided…

### PR DESCRIPTION
**Pull request by shahidmanzoorzekiexperts**
We need to pass a dynamic style to the container of SQFormGuidedWorkflow. 
currently we have <Section style={{padding: '20px'}}>
Which is fixed to 20px. In senior, we need a dynamic padding in our scenario we need it to 0px.
That's why we need a style prop here.

### CODE REVIEW

**Review by SeanGroff — CHANGES_REQUESTED** [review]
Code looks good but the comment doesn't follow the conventional commit pattern
Please follow the instructions in the readme and use the script `npm run commit`

File: src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js [hunk 1]
```diff
@@ -176,7 +177,7 @@ function SQFormGuidedWorkflow({
   });
 
   return (
-    <Section style={{padding: '20px'}}>
+    <Section style={containerStyles}>
```
Comment (20BBrown14) [inline src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:180]: How do we feel about `style={{padding: '20px', ...containerStyles}}` which will always give the 20px padding without the consumer having to define it if they want to add _other_ styles. If the consumer provides `padding` it'll override the 20px. But if they provide, for instance, `margin` our current solution will then nuke the padding which would mean the consumer would need to provide that padding as well if they want to keep it.

Plus, then we could make the default value for `containerStyles` to be `{}` which reads a bit easier and could be a bit easier to maintain in the future.

**Review by 20BBrown14 — CHANGES_REQUESTED** [review]

File: src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js (hunk 1 above)
Comment (SeanGroff) [reply src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:180]: Agreed 💯 
I didn't notice the default padding was removed.

File: src/components/SQFormGuidedWorkflow/PropTypes.js [hunk 2]
```diff
@@ -108,5 +108,7 @@ export const GuidedWorkflowProps = {
   onError: PropTypes.func.isRequired,
   /** Task Module configuration Object(s) */
   taskModules: PropTypes.arrayOf(PropTypes.shape(TaskModuleProps).isRequired)
-    .isRequired
+    .isRequired,
+  /** An object of css-in-js style properties to be passed*/
```
Comment (SeanGroff) [inline src/components/SQFormGuidedWorkflow/PropTypes.js:112]: Our Storybook depends on a properly formatted JS Doc comment.
A space is missing between passed and */

**Review by 20BBrown14 — APPROVED** [review]

**Pull request closed without merging**